spyderlib/restart_app.py

```diff
@@ -135,6 +144,29 @@ def main():
         time.sleep(0.2)  # Throttling control
 
     env = os.environ.copy()
+
+    # Reset spyder if needed
+    if reset:
+        # Build reset command
+        command_reset = '"{0}" "{1}" {2}'.format(python, spyder, args_reset)
+
+        # Try to reset
+        try:
+            p = subprocess.Popen(command_reset, shell=shell)
+            p.communicate()
+            pid_reset = p.pid
+        except Exception as error:
+            print(command)
+            print(error)
+            time.sleep(15)

```

Comment:
The idea was to be able to see the command line if an error occurs... otherwise it will close too fast... I could reduce to 10... ?